packages/client/hmi-client/src/stores/opened-workflow-node.ts

```diff
@@ -1,16 +1,28 @@
 import { defineStore } from 'pinia';
 import { ProjectAssetTypes } from '@/types/Project';
 
+interface StringValueMap {
```

Comment:
Any reason to not just use a `Map<string, number>` here?

Reply:
The problem that I run into is that if I want to edit a value in an input field `v-model` doesn't play nice when it comes to setting a Map value. I use this StringValueMap interface in various places so a lot of syntax will change when I start using .set and .get - I'll investigate this in a second pass.